en/docs/install-and-setup/upgrading-the-solution/open-banking-migration.md

```diff
@@ -0,0 +1,18 @@
+!!! tip "Before you begin:"
+    Back up all the databases before performing the migration.
+
+1. Create the `openbankingdb` table by following [Creating Databases](../setting-up-databases.md#creating-database-tables). 
```

Comment:
shall we add a note to mention that other DBs should not be touched in OB migration

Reply:
May be it would better to point out the exact script needed

Reply:
Changed to this:

> 1. Create the `openbank_openbankingdb` database and run the relevant SQL script in the `<IS_HOME>/dbscripts/open-banking/consent`
directory.

and
> shall we add a note to mention that other DBs should not be touched in OB migration

Since we are not mentioning about any other dbs here we won't need a note at this level.

Reply:
Fixed in 974508e9f4420db237b6b0beb4ca349c523a2c89